content/posts/fleek-network-node-healthchecks.md

```diff
@@ -145,47 +153,32 @@ For the ones who followed the [getting started guide](#fleek-network-getting-sta
 We execute a `cURL` request with the `--head` or `-I` flag to show the document info only, in our case the headers of our HTTP response.
 
 ```sh
-curl --head 127.0.0.1:4069
+curl --head 127.0.0.1:80
 ```
 
 Or, the shorter version:
 
 ```sh
-curl -I 127.0.0.1:4069
+curl -I 127.0.0.1:80
 ```
 
 The response is:
 
 ```sh
-HTTP/1.1 405 Method Not Allowed
-content-length: 0
-date: Wed, 04 Jan 2023 19:00:52 GMT
+HTTP/1.1 404 Not Found
+Server: nginx/1.23.3
+Date: Fri, 06 Jan 2023 18:36:09 GMT
+Content-Length: 0
+Connection: keep-alive
 ```
 
-
 We can do the same for other ports, and you'll notice different responses where for port `6009`, get an empty reply from the server because it works over a different protocol which is not HTTP/S, as described [above](#ports):
 
 ```sh
 curl: (52) Empty reply from server
 ```
 
-⚠️ A curl (52) usually means *something* accepted the TCP connection but just closed it. For our use case, we can take this as something running in port `6009`. Although, there are more appropriate ways to check this in particular.
-
-For port `8070`, you'd get:
-
-```sh
-HTTP/1.1 404 Not Found
-content-length: 0
-date: Wed, 04 Jan 2023 19:09:23 GMT
-```
-
-You can determine failure when you make a `cURL` request which fails:
-
-```sh
-curl: (7) Failed to connect to 127.0.0.1 port 4069: Connection refused
-curl: (7) Failed to connect to 127.0.0.1 port 6009: Connection refused
-curl: (7) Failed to connect to 127.0.0.1 port 8070: Connection refused
-```
```

Comment:
revert back

````suggestion
You can determine failure when you make a `cURL` request which fails:

```sh
curl: (7) Failed to connect to 127.0.0.1 port 80: Connection refused
curl: (7) Failed to connect to 127.0.0.1 port 6009: Connection refused
```
````